notifications_utils/field.py

```diff
@@ -49,9 +49,13 @@ def __repr__(self):
         return "Placeholder({})".format(self.body)
 
 
+HtmlSanitizers = Literal["strip", "escape", "passthrough", "strip_dvla_markup"]
+
+
 class Field:
+    # this needs to be made conditional so it works in the (((colour))) -> (blue) case
     placeholder_pattern = re.compile(
-        r"\({2}" r"([^()]+)" r"\){2}"  # opening ((  # body of placeholder - potentially standard or conditional.  # closing ))
+        r"\({2}" r"([^()]+)" r"\){2}"  # opening ((, body of placeholder - potentially standard or conditional, closing ))
```

Comment:
It`s a bit clearer..

Reply:
I tested locally with this regex and it seems to work globally for all test cases. It reuses the regex you added in the condition above except it adds a negative lookahead assertion to make sure that no triple parentheses are consumed but only the late beginning two parenthesis:

```suggestion
        r"\({2}" r"(?!\()" r"([\s\S]+?)" r"\){2}"
```